src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,28 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int DEFAULT_ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier(colorSupplier);
+
+        Figure[] figures = new Figure[DEFAULT_ARRAY_SIZE];
+        for (int i = 0; i < figures.length / 2; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < figures.length / 2; i++) {
```
figures.length / 2 - better make local variable

Reply:
done